src/bidiMapper/modules/network/NetworkRequest.ts

```diff
@@ -835,6 +835,10 @@ export class NetworkRequest {
       headersSize: computeHeadersSize(headers),
       bodySize: this.#bodySize,
       timings: this.#timings,
+      // @ts-expect-error CDP-specific attribute.
+      'goog:postData': this.#request.info?.request?.postData,
+      'goog:hasPostData': this.#request.info?.request?.hasPostData,
+      'goog:resourceType': this.#request.info?.type,
```

Comment:
Can you please add e2e test verifying the fields are in place

Reply:
I plan to test using Puppeteer since tests already exist there. The breakage can be caught through Puppeteer tests.

Reply:
Having existing pipeline, the Puppeteer tests can be run too late. A single test verifying the fields are in place would be enough.

Reply:
Puppeteer tests are forced on release. What is the scenario when it can it be too late?

Reply:
e.g. roll into chromedriver. Are there any issues with adding the field?

Reply:
do we plan to roll unreleased chromedriver? do we run e2e tests during the mapper roll?

Reply:
A single e2e test would not be feasible since the data is included into multiple events that include base event params, and I do not really see value in adding the tests since we already test using Puppeteer before every release.

Reply:
> do we plan to roll unreleased chromedriver? do we run e2e tests during the mapper roll?

We roll the latest revision to chromedriver.

Reply:
hm.. this sounds unexpected to me, why are not we rolling the latest released version?

Reply:
filed crbug.com/352663873

Reply:
I assume the issue with tests is resolved. If crbug.com/352663873 results in still rolling unreleased versions, we could enable Puppeteer tests on each PR. It takes 5-6 minutes unsharded and with sharding it could be down to 1-3 minutes.